sdk/ml/azure-ai-ml/azure/ai/ml/entities/_compute/_schedule.py

```diff
@@ -9,7 +9,6 @@
 from azure.ai.ml._restclient.v2022_01_01_preview.models import ComputeStartStopSchedule as RestComputeStartStopSchedule
 from azure.ai.ml._restclient.v2022_01_01_preview.models import ScheduleStatus as ScheduleState
 from azure.ai.ml._restclient.v2022_01_01_preview.models import TriggerType
-from azure.ai.ml._utils._experimental import experimental
```

Comment:
Have we confirmed for non-automl scenarios with respective team, whether we can remove experimental tag?

Reply:
Thank, reverted the non automl changes